docs/JLBP-14.md

```diff
@@ -1,29 +1,29 @@
 # [JLBP-14] Do not use version ranges
 
-- If the pom.xml of your library specifies a version range instead of a
-  single version for any particular dependency, builds at different points
-  in time can see different versions of that dependency (as your dependency
-  releases new versions), which can break your library unexpectedly.
+Always specify a single version instead of a version range for dependencies.
 
-  - Version ranges cause builds to be non-reproducible. Builds of the
-    same code with the same compiler on different machines at different
-    times can produce different artifacts with arbitrarily different behavior.
+When a pom.xml specifies a version range instead of a
+single version for any particular dependency, builds at different points
+in time can see different versions of that dependency (as your dependency
+releases new versions), which can break your product unexpectedly.
 
-  - Version ranges can pull in incomplete releases that break your
-    library's build (example:
-    https://github.com/GoogleCloudPlatform/appengine-gcs-client/issues/71).
+Version ranges cause builds to be non-reproducible. Builds of the
```

Comment:
I would combine this paragraph with the next two (starting with "Version"). When you do that, I would suggest adding words to make the list flow better, e.g. adding "also", "lastly", etc.